blog/dev_hackathon_v4_release.md

```diff
@@ -0,0 +1,134 @@
+
+<br>
+
+### PyMC v4.0 & Aesara
+
+In this [PyMC and Aesara notebook](https://github.com/juanitorduz/pymc/blob/main/docs/source/learn/core_notebooks/pymc_aesara.ipynb), Ricardo Vieira and Juan Orduz give an introduction of how PyMC models translate to Aesara graphs.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">If you want to learn more about how <a href="https://twitter.com/hashtag/PyMC?src=hash&amp;ref_src=twsrc%5Etfw">#PyMC</a> 4.0 and Aesara work together, check out this new in-depth tutorial by <a href="https://twitter.com/RicardoV944?ref_src=twsrc%5Etfw">@RicardoV944</a> and <a href="https://twitter.com/juanitorduz?ref_src=twsrc%5Etfw">@juanitorduz</a> <a href="https://t.co/7Wk4UQkqXC">https://t.co/7Wk4UQkqXC</a> <a href="https://twitter.com/AesaraDevs?ref_src=twsrc%5Etfw">@AesaraDevs</a> <a href="https://twitter.com/pymc_devs?ref_src=twsrc%5Etfw">@pymc_devs</a> <a href="https://t.co/Ro9RsGaHrf">pic.twitter.com/Ro9RsGaHrf</a></p>&mdash; Thomas Wiecki (@twiecki) <a href="https://twitter.com/twiecki/status/1534115464967819265?ref_src=twsrc%5Etfw">June 7, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<br>
+
+### Plot Conditional Adjusted Prediction
+
+The new version also brings Plot Conditional Adjusted Predictions, if you are interested in contributing we are still lacking an example of this feature in our gallery.  https://bambinos.github.io/bambi/main/examples.html
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">The new version also brings Plot Conditional Adjusted Predictions, if you are interested in contributing we are still lacking an example of this feature in our gallery. <a href="https://t.co/mXB8aXV6yN">https://t.co/mXB8aXV6yN</a> <a href="https://t.co/w9TmZnvYQd">https://t.co/w9TmZnvYQd</a> <a href="https://t.co/kOn5WQx0Zk">pic.twitter.com/kOn5WQx0Zk</a></p>&mdash; Osvaldo Martin (@aloctavodia) <a href="https://twitter.com/aloctavodia/status/1533875990992125955?ref_src=twsrc%5Etfw">June 6, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<br>
+
+## Summary [call to action]
+[*** Thomas: add a line here encouraging people to upgrade to v4.0 + link to v4 installation instructions.]
+
+### Connect with PyMC
+
+Connect with PyMC via:
+- Website:  [pymc.io](https://www.pymc.io)
+- Discourse: [discourse.pymc.io](https://discourse.pymc.io)
+- YouTube: [PyMCDevelopers](https://www.youtube.com/c/PyMCDevelopers)
+- Star GH repo:  [pymc-devs/pymc](https://github.com/pymc-devs/pymc)
+- Join Meetup: [pymc-online-meetup](https://www.meetup.com/pymc-online-meetup/)
+- Twitter: [@pymc_devs](https://twitter.com/pymc_devs)
+- LinkedIn: [@pymc](https://www.linkedin.com/company/pymc/)
+
+### Support PyMC
+
+You can support the PyMC project by making a donation to the [PyMC on Open Collective](https://opencollective.com/pymc#category-CONTRIBUTE).
```

Comment:
I think this should be https://numfocus.org/donate-to-pymc instead. But I'll let @twiecki and @fonnesbeck confirm.

Reply:
We can be consistent and use the NumFOCUS link.